spectral_cube/spectral_cube.py

```diff
@@ -2287,6 +2290,203 @@ def find_lines(self, velocity_offset=None, velocity_convention=None,
 
         return result
 
+    def reproject(self, header, order='bilinear'):
```

Comment:
Yep, but this belongs on https://github.com/radio-astro-tools/spectral-cube/pull/289